backend/routes/PostManagement/PostDeletion.py

```
@@ -0,0 +1,28 @@
+router = APIRouter()
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@router.delete("/delete_post/{post_id}")
+async def delete_post(
```

Ruff (0.8.2)

Missing return type annotation for public function `delete_post`

(ANN201)